experiments/settings_R02B10-R02B10.sh

```diff
@@ -61,7 +61,7 @@ atm_streams=(
     "output_atm_mean"
     # "output_atm_mean3d"
     "output_energy_budget"
-    "output_atm_latlon"
+    #"output_atm_latlon"
```

Comment:
```suggestion
    # "output_atm_latlon"
```
Just for consistency with the other commented out lines.